File: Dsa/CountLetterDigit.py
def CountDigitLetter(num):
    digit=0 
    letter=0
    for i in num:
        if "0"<=i<="9":
            digit+=1
        if ("a"<=i<="z") or ("A"<=i<="Z"):
            letter+=1
    return digit,letter

print(CountDigitLetter("jhfjksdhfks32423"))

File: Dsa/common/CountLetterDigit.js
function CounterDigitLetter(arr){
    let digit=0
    let letter=0
    for(i=0; i<arr.length; i++){
        let ch=arr[i]
        if(ch>="0" && ch<="9"){
            digit++
        }
        else if(ch>="a" && ch<="z"  || ch>="A" && ch<="Z"){
            letter++
        }

    }
return {digit, letter}
}
console.log(CounterDigitLetter("gduwrfsdfi241242"))